pkg/asset/manifests/azure/cluster.go

```diff
@@ -296,5 +296,5 @@ func getNextAvailableIP(ctx context.Context, installConfig *installconfig.Instal
 			}
 		}
 	}
-	return "", fmt.Errorf("failed to get available IP in given machine network: %w", err)
+	return "", fmt.Errorf("failed to get available IP in given machine network")
```

Comment:
nit: should we include a reason, for example, `no available IP` (i.e. same for line 286 above)?

I feel like `failed to ...` alone could mean some unknown internal error, which might not be descriptive enough :D

Reply:
+1 for line 299.
Currently, installer seems always running into line 286 when lack of permissions, maybe add reason what Patrick suggested for line 271?